articles/active-directory/users-groups-roles/roles-assign-graph.md

```diff
@@ -24,7 +24,7 @@ Connect to your Azure AD organization using a Global administrator account or Pr
 
```

Comment:
Should above be:

"Global administrator or Privileged Identity administrator account"

^ move the 'account' word after Privileged identity admin OR remove alltogether.